runt.toml

```diff
@@ -439,6 +439,21 @@ fud exec -s calyx.exec './target/debug/calyx' \
          grep '%Error' | sed 's/%Error: [^:]*:[^:]*:/%Error:/'
 """
 
+[[tests]]
```

Comment:
To make these tests work in CI, do we need to now run `get_hardfloat.sh` automatically before launching the tests?

Reply:
Yes! And fixed!